### PR TITLE
Emailing admin when new user signing up on site

### PR DESCRIPTION
**Pull request by supersem**
**STATUS**
READY

**DESCRIPTION**

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs

### CODE REVIEW

File: app/mailers/users_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class UsersMailer < ApplicationMailer
+  default template_path: 'mailers/users'
+
+  def send_new_user_message(current_user)
+    @user = current_user
+    email_admins = User.admins.pluck(:email)
```
Comment (igorkasyanchuk) [inline app/mailers/users_mailer.rb:6]: що стається коли нема жодного адміна?

File: app/models/user.rb [hunk 2]
```diff
@@ -53,7 +53,10 @@ class User < ApplicationRecord
          :omniauthable, omniauth_providers: %i[facebook]
 
   scope :order_desc, -> { order(:first_name, :last_name) }
+  scope :admins, -> { where(role: ADMIN) }
 
+  after_create :send_admin_mail
```
Comment (igorkasyanchuk) [inline app/models/user.rb]: перейменуй в щось типу notify_admins_about_new_user (методи мають називатись відповідно до того що роблять)

File: app/views/mailers/users/send_new_user_message.html.slim [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.email-table
```
Comment (igorkasyanchuk) [inline app/views/mailers/users/send_new_user_message.html.slim:1]: а де скріншот емейла?

**Comment by supersem** [comment]
image

File: app/views/mailers/users/send_new_user_message.html.slim (hunk 3 above)
Comment (supersem) [reply app/views/mailers/users/send_new_user_message.html.slim:1]: fix

File: app/models/user.rb (hunk 2 above)
Comment (supersem) [reply app/models/user.rb]: fix

File: app/mailers/users_mailer.rb (hunk 1 above)
Comment (supersem) [reply app/mailers/users_mailer.rb:6]: fix

**Review by igorkasyanchuk — APPROVED** [review]

**Pull request merged**